docs/mvc/tutorials/mvc-with-entity-framework/sample/src/ContosoBooks/Startup.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Diagnostics;
+using Microsoft.AspNet.Hosting;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Routing;
+using Microsoft.Framework.ConfigurationModel;
+using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.Logging;
+using Microsoft.Framework.Logging.Console;
+using ContosoBooks.Models;
+using Microsoft.Data.Entity;
+
+namespace ContosoBooks
+{
+    public class Startup
+    {
+        public Startup(IHostingEnvironment env)
+        {
+            // Setup configuration sources.
+            Configuration = new Configuration()
+                .AddJsonFile("config.json")
+                .AddEnvironmentVariables();
+        }
+
+        public IConfiguration Configuration { get; set; }
+
+        // This method gets called by the runtime.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.Configure<AppSettings>(Configuration.GetSubKey("AppSettings"));
+
+            // Add MVC services to the services container.
+            services.AddMvc();
+
+            services.AddEntityFramework()
+                //.AddInMemoryStore()
+                .AddSqlServer()
+                .AddDbContext<BookContext>(options =>
+                {
+                    options.UseSqlServer(Configuration.Get("Data:ConnectionString"));
+                });
+
+        }
+
+        // Configure is called after ConfigureServices is called.
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerfactory)
+        {
+            // Configure the HTTP request pipeline.
+
+            // Add the console logger.
+            loggerfactory.AddConsole(minLevel: LogLevel.Verbose);
+
+
+            // Add the following to the request pipeline only in development environment.
+            if (env.IsEnvironment("Development"))
+            {
+                app.UseBrowserLink();
+                app.UseErrorPage(ErrorPageOptions.ShowAll);
+            }
+            else
+            {
+                // Add Error handling middleware which catches all application specific errors and
+                // send the request to the following path or controller action.
+                app.UseErrorHandler("/Home/Error");
+            }
+
+            // Add static files to the request pipeline.
+            app.UseStaticFiles();
+
+            // Add MVC to the request pipeline.
+            app.UseMvc(routes =>
+            {
+                routes.MapRoute(
+                    name: "default",
+                    template: "{controller=Home}/{action=Index}/{id?}");
+            });
+            SampleData.Initialize(app.ApplicationServices);
```

Comment:
This is ok for a demo app, but if the idea is to show folks how to use EF in a real application then we should be using migrations